stl/inc/tuple

```diff
@@ -791,6 +957,14 @@ _CONSTEXPR20 void swap(tuple<_Types...>& _Left, tuple<_Types...>& _Right) noexce
     return _Left.swap(_Right);
 }
 
+#if _HAS_CXX23
+template <class... _Types, enable_if_t<conjunction_v<_STD _Is_swappable<const _Types>...>, int> = 0>
+constexpr void swap(const tuple<_Types...>& _Left, const tuple<_Types...>& _Right) noexcept(

```

Comment:
This is partially pre-existing (copying the overload above), but as this is non-member `swap()`, we don't need the extra `_STD` qualification to defend against class-member lookup:
https://github.com/microsoft/STL/blob/39332e104d5274b8b1f48ebe380ba1dfc7ce4cc0/stl/inc/tuple#L157-L160